go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/tinylib/msgp v1.1.5
 	github.com/uber/jaeger-lib v2.4.0+incompatible
+	github.com/xdg-go/scram v0.0.0-20180814205039-7eeb5667e42c
```

Comment:
How ok are we to depend on a very old, not actively maintained package?

Does not even have go[mod/sum].

Reply:
Honestly? It doesn't worry me that much, as the code is fairly simple and based on a couple of RFCs. Once the standard is implemented, there's not much else to do. I do have thought about the responsiveness of the author in that repository (see the issue opened by @yurishkuro) but I don't see a reason for implementing this ourselves at this moment.